source/base/exceptions.cc

```diff
@@ -336,6 +343,45 @@ void ExceptionBase::generate_message () const
 
 
 
+ExcMPI::ExcMPI (const int error_code)
+  :
+  error_code (error_code)
+{}
+
+void ExcMPI::print_info (std::ostream &out) const
+{
+#ifdef DEAL_II_WITH_MPI
+  char error_name[MPI_MAX_ERROR_STRING];
+  error_name[0] = '\0';
+  int resulting_length = MPI_MAX_ERROR_STRING;
+  // get the string name of the error code by first converting it to an error
+  // class.
+  int error_class = 0;
+  // there is nothing we can do if these functions fail (they just provide
+  // better input), so assume that the given error code is valid.
```

Comment:
Output? 

I don't understand what these two lines comment on.  On the fact that the following mpi calls are not return value checked?